src/stub_logger.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+/* eslint-env goodeggs/server-side-test */
+import assert from 'assert';
+
+export default function stubLogger (logger: Object, sandbox: Object) {
```

Comment:
I feel like this signature is backwards; arguments to a function should be ordered most generic to least such that you can use partial application to compose more specific versions of a function.

(I'm not sure that partialing _here_ has tons of utility, but thematically I feel pretty strongly about this.)

Reply:
How about defining interface types for `logger` and `sandbox` to get better type safety?

Reply:
Also fixed in the new interface. Yes, the logger type could be more strict. We still don't have a good pattern for sharing types like this. What do you recommend?

Reply:
I think you want something like this:

```js
interface Logger {
  trace(...*) => mixed, 
  debug(...*) => mixed,
  info(...*) => mixed,
  warn(...*) => mixed,
  error(...*) => mixed,
  fatal(...*) => mixed,
  child(...*) => mixed,
}
```